app/code/Magento/Catalog/Api/ProductRenderListInterface.php

```diff
@@ -25,8 +25,14 @@ interface ProductRenderListInterface
      * @param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
      * @param int $storeId
      * @param string $currencyCode
+     * @param int $customerGroupId
      * @return \Magento\Catalog\Api\Data\ProductRenderSearchResultsInterface
      * @since 102.0.0
      */
-    public function getList(\Magento\Framework\Api\SearchCriteriaInterface $searchCriteria, $storeId, $currencyCode);
+    public function getList(
+        \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria,
+        $storeId,
+        $currencyCode,
+        $customerGroupId = null
```

Comment:
Such code change is prohibited according to https://developer.adobe.com/commerce/contributor/guides/code-contributions/backward-compatibility-policy/#adding-parameters-in-public-methods

Is there a way to obtain customer group of logged in user maybe instead of passing a new parameter?